### ISSUE TITLE
get_coupling_matrix for NmtWorkspaceFlat

### ISSUE THREAD

**Issue by xzackli** [LSSTDESC/NaMaster#134]
It looks like the flat workspace is missing the counterpart to https://github.com/LSSTDESC/NaMaster/blob/1c0371dd884910345a5c66d40007f56f29a0e982/pymaster/workspaces.py#L102 which may be helpful for demonstrations.

**Comment by martinjameswhite** [comment]
There also does not appear to be a flat-sky equivalent to the very useful "get_bandpower_windows" method of the full-sky workspace.

**Comment by damonge** [comment]
Right, @xzackli @martinjameswhite , good point. The trouble here is that the "true" space in flat sky is a continuum (unlike on the sphere, where multipoles are discrete), so one would need to come up with a reasonable sampling rate for ell. Because of this, and because at the time no one seemed interested in having these, I never got around to implementing any of these functions.

I could pick a sampling (e.g. delta_ell = 2pi/L_max for the map in question) or (perhaps more complicated), this could be decided by the user. Thoughts?

**Comment by martinjameswhite** [comment]
From my perspective the most common use for a bandpower window function is that I multiply some "theory" Cell by the window to get the "observed" Cobs.  In this case I'd want to get out an Nband x Nellmax matrix with integer spaced ells.  I agree this isn't as cleanly defined as in the spherical sky case, but that would be my suggestion.
